ui/src/app/views/table.html

```diff
@@ -1,22 +1,141 @@
+
 <div class="table-responsive-vertical md-whiteframe-z1">
     <table id="table" class="table table-hover table-bordered">
         <thead>
         <tr>
             <th>#</th>
             <th>User</th>
-            <th>Download name</th>
-            <th>Size</th>
+	    <th>Download name</th>
+            <th>More</th>
             <th>Completed time</th>
         </tr>
         </thead>
         <tbody>
+
         <tr ng-repeat="data in downloads">
+
+		
             <td data-title="ID">{{$index + 1}}</td>
-            <td data-title="User">{{data.user_name}}</td>
-            <td data-title="Issue">{{data.download_name}}</td>
-            <td data-title="Progress">{{data.size}}</td>
-            <td data-title="Completed time" am-time-ago="data.completed_time|amFromUnix"</td>
+            <td data-title="User">{{data.user_name}} </td>
+	  <td class="part" data-title="Issue">{{data.download_name}}</td>
+	    <td data-title="More"><div class="dropdown">
+  <button class="dropbtn">Details</button>
+  <div class="dropdown-content">
+    <li class="edit"><i class="fa fa-file"></i> FileSize : {{data.size}}</l1>
+    <li class="edit"><i class="fa fa-cloud-download"></i> Downloaded On : {{ (data.completed_time*1000) | date : "h:mm a d MMM, y " }}</li>
+    <li onclick="remove()" id="remove" class="edit"><i class="fa fa-trash"></i> Remove</li>
+  </div>
+</div></td>
+            <td data-title="Completed time" am-time-ago="data.completed_time|amFromUnix"></td>
         </tr>
         </tbody>
     </table>
 </div>
+<style>
```

Comment:
use these in style sheets
https://github.com/scorelab/Bassa/blob/master/ui/src/app/stylesheets/_table.scss